.env.example

```
@@ -70,6 +74,21 @@ MYSQL_PASSWORD=task_password
+# =============================================================================
+# SHARING CONFIGURATION
+# =============================================================================
+
+# Team/Task sharing URLs (IMPORTANT: Update these for production deployment!)
+# These URLs are used when generating share links for teams and tasks
+# - Local development: http://localhost:3000
+# - Server deployment: http://your-domain.com or https://your-domain.com
+# TEAM_SHARE_BASE_URL is the base URL for team sharing (includes /chat path)
+TEAM_SHARE_BASE_URL=http://localhost:3000/chat
+# TASK_SHARE_BASE_URL is the base URL for task sharing
+TASK_SHARE_BASE_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TASK_SHARE_BASE_URL key should go before the TEAM_SHARE_BASE_URL key

(UnorderedKey)